src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(({ sex }) => sex === 'm');
+
+  const filteredByCentury = men.filter(
+    ({ died }) => Math.ceil(died / 100) === century
+  );
```

Comment:
what if century is undefined?
you don't have to make filter by century if you don't have century

think about how to combine this logic in the first filter